src/components/Tooltip/Tooltip.js

```diff
@@ -452,83 +451,58 @@ class Tooltip extends Component {
       `${prefix}--tooltip__label`,
       triggerClassName
     );
-    const ariaOwnsProps = !open
-      ? {}
-      : {
-          'aria-owns': tooltipId,
-        };
 
-    const ariaDescribedbyProps = !open
-      ? {}
-      : {
-          'aria-describedby': tooltipId,
-        };
-    const finalIcon = IconCustomElement ? (
-      <IconCustomElement
-        name={iconName}
-        aria-labelledby={triggerId}
-        aria-label={iconDescription}
-        ref={mergeRefs(ref, node => {
-          this.triggerEl = node;
-        })}
-      />
-    ) : (
-      <Icon
-        icon={!icon && !iconName ? iconInfoGlyph : icon}
-        name={iconName}
-        description={iconDescription}
-        iconTitle={iconTitle}
-        iconRef={mergeRefs(ref, node => {
-          this.triggerEl = node;
-        })}
-      />
-    );
+    const iconProperties = { name: iconName, role: null, description: null };
+
+    const properties = {
+      role: 'button',
+      tabIndex: tabIndex,
+      onClick: this.handleMouse,
+      onKeyDown: this.handleKeyPress,
+      onMouseOver: this.handleMouse,
+      onMouseOut: this.handleMouse,
+      onFocus: this.handleMouse,
+      onBlur: this.handleMouse,
+      ref: mergeRefs(ref, node => {
```

Comment:
This `ref` should be set to `<div className={triggerClasses}>` _only_ when its content is text-only. This is for locating tooltip relative to icon when it's present, and to the containing `<div>` otherwise.

Reply:
thanks for explaining, was not very clear to me .. will fix that